src/mock/point.js

```diff
@@ -0,0 +1,26 @@
+import { getRandomInteger } from '../util';
+import dayjs from 'dayjs';
+
+const generateDate = () => {
+  const maxDaysGap = 2000;
+  const daysGap = getRandomInteger(-maxDaysGap, maxDaysGap);
+
+  return dayjs().add(daysGap, 'm').toDate();
+};
+
+export const generatePoint = () => {
+  const dateFrom = generateDate();
+  let dateTo = generateDate();
+  while (dateTo <= dateFrom) {
+    dateTo = generateDate();
+  }
```

Comment:
Циклы и условия обычно отделяют пробелами 
```suggestion

    while (dateTo <= dateFrom) {
    dateTo = generateDate();
  }
  
```